lib/ngtcp2_macro.h

```diff
@@ -31,8 +31,6 @@
 
 #include <stddef.h>
 
-#include <ngtcp2/ngtcp2.h>
-
```

Comment:
Could you leave this line and just delete the macro?

Reply:
Absolutely, I'll update shortly. Thanks